core/adapters/eth_tx_internal_test.go

```diff
@@ -0,0 +1,160 @@
+package adapters
```

Comment:
This is an intentional whitebox test, I think it makes sense for this case and I think we could continue with the pattern: xxx_test.go (blackbox test) xxx_internal_test.go (whitebox test)